lib/LLVMIRCodeGen/FunctionSpecializer.cpp

```diff
@@ -27,6 +26,7 @@
 #include "llvm/Transforms/Utils/Cloning.h"
 #include "llvm/Transforms/Utils/ValueMapper.h"
 
+#define DEBUG_TYPE "ir-function-specializer"
```

Comment:
I had to move this.  When I remerged, I got build errors for undefined DEBUG_TYPE when expanding the STATISTIC macro.  Not sure if this is an LLVM 7 vs. 6 thing, or something else...but I do know some LLVM headers #undef DEBUG_TYPE.

Reply:
This is very strange.